src/content/security/index.md

```diff
@@ -254,6 +254,12 @@ Some things to remember:
 
 [Man loses $200k in mining pool scam](https://www.reddit.com/r/CoinBase/comments/r0qe0e/scam_or_possible_incredible_payout/)
 
+### "Eth2" token scam {#eth2-token-scam}
+
+With the merge coming in 2022, scammers have taken advantage of the confusion around the term "Eth2" to try and get users to redeem their ETH for an "ETH2" token. There is no "ETH2" or any other new token introduced with the merge. The ETH that you own today will continue to be the same ETH after the merge, and there is no need to do any swaps of your ETH for the merge.
```

Comment:
```suggestion
With [the merge](/upgrades/merge/) coming in 2022, scammers have taken advantage of the confusion around the term "Eth2" to try and get users to redeem their ETH for an "ETH2" token. There is no "ETH2" or any other new token introduced with the merge. The ETH that you own today will continue to be the same ETH after the merge, and there is no need to do any swaps of your ETH for the merge.
```

Reply:
We should add a link to the /upgrades/ homepage section that explains what happened to Eth2.